<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ZCBLRLP0293" failures="1" tests="19" name="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" time="238.945" errors="2" timestamp="2024-07-09T16:01:55 IST" skipped="0">
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verify_NonEditable_LocationField" time="7.497"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verify_NonEditable_DateField" time="7.379"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verify_NonEditable_TimeField" time="6.448"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verify_EarnPoints_Navigation_From_MeberDetailsScreen" time="7.648"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verify_MemberDetails_Content_Displaying_On_EarnPointScreen" time="37.852">
    <error message="class org.openqa.selenium.By$ByXPath cannot be cast to class org.openqa.selenium.WebElement (org.openqa.selenium.By$ByXPath and org.openqa.selenium.WebElement are in unnamed module of loader &#039;app&#039;)" type="java.lang.ClassCastException">
      <![CDATA[java.lang.ClassCastException: class org.openqa.selenium.By$ByXPath cannot be cast to class org.openqa.selenium.WebElement (org.openqa.selenium.By$ByXPath and org.openqa.selenium.WebElement are in unnamed module of loader 'app')
at zapcg.Capillary.PageObject.earnPointsPage.verifyTheMemberDetailsContentDisplaying(earnPointsPage.java:192)
at zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases.verify_MemberDetails_Content_Displaying_On_EarnPointScreen(EarnPoint_Valid_TestCases.java:86)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </error>
  </testcase> <!-- verify_MemberDetails_Content_Displaying_On_EarnPointScreen -->
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verify_HeaderHyperlink_On_EarnPoint_Page" time="6.991"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verify_BackButton_Navigation_On_EarnPoint_Page" time="7.278"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verify_Hamburger_MemberLookupOption_From_EarnPoint_Page" time="8.672"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verify_Hamburger_LocationOption_From_EarnPoint_Page" time="8.179"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verify_HamBurger_LogoutOption_From_EarnPoint_Page" time="8.226"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verify_Date_Equal_To_System_CurrentDate" time="7.649"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verify_Time_Equal_To_System_CurrentTime" time="7.060"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verify_With_Empty_TransactionAmount" time="7.908"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="empty_TxnAmount_But_GoToPassPoint_Value" time="7.790"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verifySuccessMessageForOnlyTransactionAmount" time="10.481"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verify_TaxAssumedAmount" time="7.181"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verify_SuccessMessage_For_RedeemPoint" time="67.958">
    <failure message="An error occurred while verifying the success message: Expected condition failed: waiting for visibility of Proxy element for: DefaultElementLocator &#039;By.xpath: //p[@id=&#039;EPMDLMSG&#039; and contains(text(), &#039;transaction has been successfully submitted. They have earned 35 points and have spent 1 points.&#039;)]&#039; (tried for 10 second(s) with 500 milliseconds interval)" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: An error occurred while verifying the success message: Expected condition failed: waiting for visibility of Proxy element for: DefaultElementLocator 'By.xpath: //p[@id='EPMDLMSG' and contains(text(), 'transaction has been successfully submitted. They have earned 35 points and have spent 1 points.')]' (tried for 10 second(s) with 500 milliseconds interval)
at org.testng.Assert.fail(Assert.java:111)
at zapcg.Capillary.PageObject.earnPointsPage.verifySuccessMessageIfRedeemingPoints(earnPointsPage.java:765)
at zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases.verify_SuccessMessage_For_RedeemPoint(EarnPoint_Valid_TestCases.java:232)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </failure>
  </testcase> <!-- verify_SuccessMessage_For_RedeemPoint -->
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verify_Eligible_EarningPoints" time="7.896"/>
  <testcase classname="zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases" name="verify_ExceedThresold_SuccessMessage" time="8.852">
    <error message="Cannot invoke &quot;org.openqa.selenium.SearchContext.findElement(org.openqa.selenium.By)&quot; because &quot;this.searchContext&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.SearchContext.findElement(org.openqa.selenium.By)" because "this.searchContext" is null
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:68)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at com.sun.proxy.$Proxy18.sendKeys(Unknown Source)
at zapcg.Capillary.PageObject.memberLookupPage.enterMemberId(memberLookupPage.java:94)
at zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases.verify_ExceedThresold_SuccessMessage(EarnPoint_Valid_TestCases.java:288)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </error>
  </testcase> <!-- verify_ExceedThresold_SuccessMessage -->
</testsuite> <!-- zapcg.Capillary.EarnPointPageTestCases.EarnPoint_Valid_TestCases -->
